src/main/java/nourl/mythicmetals/MythicMetals.java

```diff
@@ -50,14 +49,15 @@
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
+import java.util.Arrays;
 import java.util.UUID;
 
 public class MythicMetals implements ModInitializer, EntityComponentInitializer {
     public static Logger LOGGER = LogManager.getLogger();
     public static final String MOD_ID = "mythicmetals";
     public static final int CONFIG_VERSION = 11;
 
-    public static final AbstractMinecartEntity.Type BANGLUM_TNT = ClassTinkerers.getEnum(AbstractMinecartEntity.Type.class, "BANGLUM_TNT");
+    public static final AbstractMinecartEntity.Type BANGLUM_TNT = Arrays.stream(AbstractMinecartEntity.Type.class.getEnumConstants()).filter(constant -> constant.name().equals("BANGLUM_TNT")).toList().get(0);
```

Comment:
While beautifully hacky, this method can probably be replaced by a `Enum.valueOf()` call.

Reply:
I was proud of that one, too.